### DISCUSSION TITLE
please answer

### DISCUSSION THREAD

**Discussion by Cqban** [STRINGLABITSOLUTIONS/Bhojon#358] [Q&A] [answered]
answer for the badge please

**Comment by fcasamento** [comment]
Hello @Cqban friend. What do you need?

**Reply by Preetiraj3697** [reply]
Accepted my answer.

**Reply by fcasamento** [reply]
@Preetiraj3697  Of course, be the first person to respond to a thread and accept it as an answer.

**Reply by fcasamento** [reply]
For example, this question from @Cqban I answered first, but he still hasn't marked my answer as the one that "solves" this question. @Preetiraj3697  would have to do the same as me and wait for someone to mark it as answered.

**Comment by subidit** [comment, marked as answer]
Mark a reply as answered by clicking the tick ✔︎ button on the top right corner.

**Reply by muhammadnurulahsan** [reply]
How do I get the Galaxy Brain badge?

**Reply by Preetiraj3697** [reply]
@muhammadnurulahsan  Yes, discussions in any repository, and your answers need to be marked as solutions. You need a certain count for each level.

Unfortunately there's recently a lot of spam here by people who post minimal "questions" and then select useless answers to get the achievement (like this thread). Anyone who's that desperate for the achievement should at least do it in their own repository where they don't bother others.

**Comment by Guardiaans** [comment]
Answered. Could you mark as answered?

**Reply by muhammadnurulahsan** [reply]
How do I get the Galaxy Brain badge?

**Reply by Preetiraj3697** [reply]
@muhammadnurulahsan Yes, discussions in any repository, and your answers need to be marked as solutions. You need a certain count for each level.

Unfortunately there's recently a lot of spam here by people who post minimal "questions" and then select useless answers to get the achievement (like this thread). Anyone who's that desperate for the achievement should at least do it in their own repository where they don't bother others.

**Comment by ChrisHagedorn** [comment]
Sure!

**Comment by dnya0** [comment]
Good!

**Comment by muhammadnurulahsan** [comment]
How do I get the Galaxy Brain badge?

**Reply by Preetiraj3697** [reply]
Yes, discussions in any repository, and your answers need to be marked as solutions. You need a certain count for each level.

Unfortunately there's recently a lot of spam here by people who post minimal "questions" and then select useless answers to get the achievement (like this thread). Anyone who's that desperate for the achievement should at least do it in their own repository where they don't bother others.

**Reply by ankit-akash** [reply]
> How do I get the Galaxy Brain badge?
@muhammadnurulahsan  If you need Galaxy brain badge then answer 2 of the following questions and I will mark them correct. After 2nd correct marked answer you will be getting a galaxy brain badge.Link-
https://github.com/ankit-akash/DSA/discussions

**Comment by ankit-akash** [comment]
IF you need Galaxy brain badge then answer 2 of  the following questions and I will mark them correct. After 2nd correct marked answer you will be getting a galaxy brain badge.  Link- https://github.com/ankit-akash/DSA/discussions

**Reply by Preetiraj3697** [reply]
I answer it @ankit-akash

**Comment by AlefMemTav** [comment]
ok
